packages/preferences/src/browser/preferences-json-schema-contribution.ts

```diff
@@ -38,6 +39,8 @@ export class PreferencesJsonSchemaContribution implements JsonSchemaContribution
     @inject(WorkspaceService)
     protected readonly workspaceService: WorkspaceService;
 
+    protected readonly debouncedUpdateInMemoryResources = debounce(() => this.updateInMemoryResources(), 200);
```

Comment:
I'd prefer to load the "magic" debounce number from a constant at the top of the file. WDYT?